src/server/accessors/Notifier.ts

```diff
@@ -46,4 +71,27 @@ export class Notifier implements INotifier {
     public getMessageBuilder(): IMessageBuilder {
         return new MessageBuilder();
     }
+
+    /**
+     * Creates a new direct message room between two users.
+     *
+     * @param {IUser} user - The first user to create the room for.
+     * @param {IUser} sender - The second user to create the room for.
+     * @returns {Promise<IRoom>} A Promise that resolves with the newly created direct message room.
+     */
+    private async createDMRoom(user: IUser, sender: IUser): Promise<IRoom> {
+        const newDMRId = await this.roomBridge.doCreate(
+            {
+                type: RoomType.DIRECT_MESSAGE,
+                usernames: [user.username, sender.username],
+                creator: sender,
+                id: '',
+                slugifiedName: '',
```

Comment:
Same here

Reply:
It is required, or TS errors pop up.

Reply:
🤦🏼‍♂️ damn you typescript sometimes ...

could you add a small comment to clear that up? Might feel unnecessary but when reading the code my first thought was 'what is the significance of passing empty strings here' - while there wasn't any

Reply:
Don't worry about it, just trying to ease up our own code jumps when reading in the future :)